tests/Console/ConfigurationResolverTest.php

```diff
@@ -988,8 +981,10 @@ public function testResolveRulesWithOption()
 
     public function testResolveRulesWithUnknownRules()
     {
-        $this->setExpectedException(
-            \PhpCsFixer\ConfigurationException\InvalidConfigurationException::class,
+        $this->expectException(
+            \PhpCsFixer\ConfigurationException\InvalidConfigurationException::class
```

Comment:
Quick note: it appears that `PhpCsFixer\ConfigurationException\InvalidConfigurationException` has already been imported, so while this out of scope of the changes suggested in this PR, what do you think about simplifying the FQCN here, as well as putting the invocation on a single line?

Reply:
doing it in this one place is pointless, we already have a future request to import all usages like here, that would fix issue globally